### ISSUE TITLE
Emulator Test

### ISSUE THREAD

**Issue by nermin-yehia** [NorthBit/Metaphor#12]
Hi
can i try that code using emulator .. instead of real Nexus 5 device ?

**Comment by xstation-vasilii** [comment]
yes, I have done that ,you must modify some code according to your target emulator.It‘s not easy if you do not understand Metaphor.

**Comment by nermin-yehia** [comment]
thanks a lot
i bought Nexus 5 to test on it.
